Source/WTF/wtf/PlatformEnable.h

```diff
@@ -1030,6 +1030,14 @@
 #error "ENABLE(PREDEFINED_COLOR_SPACE_DISPLAY_P3) requires ENABLE(DESTINATION_COLOR_SPACE_DISPLAY_P3)"
 #endif
 
+#if (PLATFORM(MAC) \
+    || (PLATFORM(IOS) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 170400) \
+    || PLATFORM(GTK) \
+    || PLATFORM(WPE) \
+    || PLATFORM(WIN))
+#define ENABLE_WEBPROCESS_CACHE 1
```

Comment:
This is okay, but I've learned that the intended behavior would be to have the MAC and IOS bits set in `PlatformEnableCocoa.h`, the WIN bit in `PlatformEnableWinCairo.h` and this file take the form:

```
#if !defined(ENABLE_WEBPROCESS_CACHE)
#define ENABLE_WEBPROCESS_CACHE 0
#endif
```

This doesn't work perfectly since GTK/WPE does not have a dedicated PlatformEnablement file at present, which messes up this format.

I suggest you do the following:

1. Put the MAC/IOS enablement in `PlatformEnableCocoa.h`
2. Put the WIN enablement in `PlatformEnableWinCairo.h`
3. Have this section become this:

```
#if !defined(ENABLE_WEBPROCESS_CACHE) && (PLATFORM(GTK) || PLATFORM(WPE))
#define ENABLE_WEBPROCESS_CACHE 1
#endif

#if !defined(ENABLE_WEBPROCESS_CACHE)
#define ENABLE_WEBPROCESS_CACHE 0
#endif
```

Then we (rather, *I* or Igalia people) can clean up the GTK stuff in the future

Reply:
Ah, that is a good point! Fixed in latest patch.

Thanks for reviewing!